framework/db/Connection.php

```diff
@@ -687,12 +712,17 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
+        $driver = $this->getDriverName();
+        $config = ['class' => 'yii\db\Command'];
+        if ($this->commandClass !== $config['class']) {
```

Comment:
Looks suspicious. Have you meant to do this check after `if (isset($this->commandMap[$driver])) {`?